doc/src/examples/arithmetic/cross_product

```diff
@@ -0,0 +1,47 @@
+
+// Boost.Geometry
+// QuickBook Example
+
+// Copyright (c) 2020, Aditya Mohan
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+// Calculate the cross product of two points
+
+
```

Comment:
you have to add doxygen commands in this file for the documentation to be generated. Have a look at how e.g. algorithms are documented. You may also consult this https://github.com/boostorg/geometry/wiki/Generating-and-Improving-Documentation